app/assets/javascripts/discourse/templates/user/summary.hbs

```diff
@@ -77,10 +77,11 @@
         <ul>
           {{#each model.links as |link|}}
             <li>
+              {{!-- template-lint-disable  --}}
               <a class='domain'
                  href='{{unbound link.url}}'
                  title='{{unbound link.title}}'
-                 rel='{{unless user.removeNoFollow 'nofollow ugc noopener'}}'
+                 rel='{{unless user.removeNoFollow 'nofollow ugc noopener' 'noopener'}}'
```

Comment:
This should work, right?

```suggestion
                 rel='noopener {{unless user.removeNoFollow 'nofollow ugc'}}'
```

Reply:
(Same with the other `template-lint-disable` instances)

Reply:
yes I think it would, will try :+1:

Reply:
Sadly it still errors in the linting, but I changed it to your syntax as it still prevents duplicating noopener